lua/nvim-gps/init.lua

```diff
@@ -322,10 +322,14 @@ function M.get_data()
 	local node = current_node
 
 	local function add_node_data(pos, capture_name, capture_node)
+		local text = vim.treesitter.query.get_node_text(capture_node, 0)
+		if type(text) == table then
```

Comment:
according to [this](https://github.com/nvim-treesitter/nvim-treesitter/pull/2801#issuecomment-1101555994) comment, the new function does not return a table, instead returns the whitespace characters as it is. So these characters need to be replaced by a single space character.